src/components/lib/VDropdown/VDropdown.vue

```diff
@@ -102,7 +104,14 @@ watch(
         </span>
       </span>
       <span v-else class="placeholder">{{ placeholder }}</span>
-      <div class="arrow"></div>
+      <img
+        :src="arrowIcon"
```

Comment:
can do `src="@/assets/iconography/arrow.svg"` directly instead of importing it

Reply:
Resolved